Node/platformio.ini

```diff
@@ -0,0 +1,101 @@
+;PlatformIO Project Configuration File
+;
+;   Build options: build flags, source filter
+;   Upload options: custom upload port, speed and extra flags
+;   Library options: dependencies, extra library storages
+;   Advanced options: extra scripting
+;
+; Please visit documentation for the other options and examples
+; https://docs.platformio.org/page/projectconf.html
+
+[platformio]
+src_dir = ./
+include_dir = Common
+
+[env:DTS-SAMD]
+platform = atmelsam
+board = sodaq_autonomo
+framework = arduino
+src_filter = 
+    +<DTS>
+build_flags = 
+    -I./DTS/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:DTS-Uno]
+platform = atmelavr
+board = uno
+framework = arduino
+src_filter = 
+    +<DTS>
+build_flags = 
+    -I./DTS/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:Vacuum-SAMD]
+platform = atmelsam
+board = sodaq_autonomo
+framework = arduino
+src_filter = 
+    +<Vacuum>
+build_flags = 
+    -I./Vacuum/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:Vacuum-Uno]
+platform = atmelavr
+board = uno
+framework = arduino
+src_filter = 
+    +<Vacuum>
+build_flags = 
+    -I./Vacuum/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:WindTunnel-SAMD]
+platform = atmelsam
+board = sodaq_autonomo
+framework = arduino
+src_filter = 
+    +<WindTunnel>
+build_flags = 
+    -I./WindTunnel/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:WindTunnel-Uno]
+platform = atmelavr
+board = uno
+framework = arduino
+src_filter = 
+    +<WindTunnel>
+build_flags = 
+    -I./WindTunnel/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:Examples]
+platform = atmelavr
+board = uno
+framework = arduino
+src_filter = 
+    +<Examples>
+build_flags = 
+    -I./Examples/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
+
+[env:Test]
+platform = atmelavr
+board = uno
+framework = arduino
+src_filter = 
+    +<Test>
+build_flags = 
+    -I./Test/
+    -I./Common/drivers/inc
+    -I./Common/sensors/inc
```

Comment:
Does the platform for testing have to be native?

Reply:
Yes, fixed and changed Test to Testing